cuda/Simulation.cu

```diff
@@ -13,10 +12,12 @@
 // line argument.
 ////////////////////////////////////////////////////////////////////////////////////
 
-unsigned long long run_event_based_simulation_baseline(Inputs in, SimulationData SD, int mype)
+unsigned long long run_event_based_simulation_baseline(Inputs in, SimulationData SD, int mype, Profile* profile)
 {
+	double start = 0.0;
         // Move Data to GPU
         SimulationData GSD = move_simulation_data_to_device(in, mype, SD);
+	profile->h2d_time = get_time() - start;
```

Comment:
Spelling out `host_to_device` might be clearer than using an abbreviated name (particularly if we want this to eventually get merged in upstream)

Reply:
Will do